src/renderer/scss/component/_snack-bar.scss

```diff
@@ -1,30 +1,54 @@
 .snack-bar {
-  padding: $spacing-vertical;
-  position: fixed;
-  top: $spacing-vertical;
+  background-color: var(--color-primary);
```

Comment:
You should create a variable for this `--snack-bar-bg` `--snack-bar-color`

So we can change them for dark mode. Similar to how we do it with buttons:
https://github.com/lbryio/lbry-app/blob/master/src/renderer/scss/_vars.scss#L93